articlequality/feature_lists/enwiki.py

```diff
@@ -15,18 +15,18 @@
 
 # Templates
 infobox_templates = wikitext.revision.template_names_matching(
-    r"infobox", name="enwiki.revision.infobox_templates")
+    r"infobox$", name="enwiki.revision.infobox_templates")
 CN_TEMPLATES = [
     r"Citation[_ ]needed",
     r"Cn",
     r"Fact"
 ]
 cn_templates = wikitext.revision.template_names_matching(
-    "|".join(CN_TEMPLATES), name="enwiki.revision.cn_templates")
+    "$|".join(CN_TEMPLATES)+"$", name="enwiki.revision.cn_templates")
```

Comment:
Have you considered a regex in the format `(?:foo|bar|baz)$`, with a single `$` after the parenthesis?

Reply:
okay I'll try to check that

Reply:
Oh, not really. I was thinking in something like this:
`(?:Citation[_ ]needed|Cn|Fact)$`
See https://regex101.com/r/GinCwl/1.

Reply:
> Oh, not really. I was thinking in something like this: `(?:Citation[_ ]needed|Cn|Fact)$` See https://regex101.com/r/GinCwl/1.

Oh that does seem much better, thank you. 
I'll do that and run some tests